misc/python/materialize/parallel_workload/database.py

```diff
@@ -296,7 +296,8 @@ def __init__(
                     "ON COMMIT",
                     f"EVERY '{rng.randint(1, 60)} seconds {rng.randint(0, 60)} minutes'",
                     f"EVERY '{rng.randint(1, 60)} seconds {rng.randint(0, 60)} minutes' ALIGNED TO (mz_now())",
-                    f"AT mz_now()::string::int8 + {rng.randint(0, 3600000)}",
+                    # Always in the future of all refreshes of previously generated MVs
+                    "AT mz_now()::string::int8 + 1000",
```

Comment:
Would
```
f"AT mz_now()::string::int8 + {rng.randint(1000, 3600000)}",
```
also have worked?

Reply:
Unfortunately not. We'd have to keep track of the actual timestamp used in the MVs another MV depends on to make sure the dependency MV never has a ts more in the future than the dependent one.